rest/src/main/java/discord4j/rest/route/Routes.java

```diff
@@ -29,7 +29,7 @@ public abstract class Routes {
      * @see <a href="https://discord.com/developers/docs/reference#base-url">
      * https://discord.com/developers/docs/reference#base-url</a>
      */
-    public static final String BASE_URL = "https://discord.com/api/v9";
+    public static final String BASE_URL = "https://discord.com/api/v10";
```

Comment:
This change not break any more than the intent?

Reply:
There are other changes but already supported with the current version of D4J (as the gateway already uses the latest version)